awx/ui_next/src/components/JobList/JobListItem.jsx

```diff
@@ -202,9 +209,29 @@ function JobListItem({
                   dataCy={`job-${job.id}-project`}
                 />
               )}
-              <ExecutionEnvironmentDetail
-                virtualEnvironment={job.custom_virtualenv}
-                executionEnvironment={execution_environment}
+              <Detail
```

Comment:
Is there any reason we can't move this logic/tooltip into the ExecutionEnvironmentDetail and trigger it via a prop? It's seems strange not use that component, especially when we have to repeat this code again in the JobDetail page

Reply:
@keithjgrant, I will take a look.

Reply:
@keithjgrant updated per suggestion.